paasta_tools/mesos/cluster.py

```diff
@@ -15,34 +15,30 @@
 # limitations under the License.
 import asyncio
 import itertools
-from typing import Set
 
 from . import exceptions
 from paasta_tools.utils import paasta_print
 
-missing_slave: Set[str] = set()
-
 
 async def get_files_for_tasks(task_list, file_list, max_workers):
     no_files_found = True
 
     async def process(task_fname):
         task, fname = task_fname
         try:
-            fobj = await task.file(fname)
+            fobj = await task.file(task_fname)
```

Comment:
I think this will break -- `task_fname` is a tuple of `(task, fname)`

Reply:
fixed